json-ld-recharter/index.html

```diff
@@ -296,70 +301,113 @@ <h2>YAML-LD</h2>
    </ul>
   </section>
 
-  <section class="slide" id="yaml-ld-human-readable">
-    <h2>JSON-LD is…</h2>
+  <section class="slide" id="yaml-ld-example">
+    <h2>Utopia Driver's License VC<br/><small>in YAML-LD!</small></h2>
+    <pre style="font-size: 40%;"><code class="language-yaml">{
```

Comment:
```suggestion
    <pre style="font-size: 40%;"><code class="language-json">{
```

Did this one change for a reason?

Reply:
I figured out the point you were making. 😉